src/Compilers/Extension/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="7922692f-f018-45e7-8f3f-d3b7c0262841" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Compilers</DisplayName>
     <Description xml:space="preserve">Package of the Roslyn compilers that enables them for Visual Studio builds.</Description>
+    <License>Eula.rtf</License>
```

Comment:
📝 It seems like the case here should match the file referenced in the targets file

```suggestion
    <License>EULA.rtf</License>
```

Reply:
Updated this in all locations. Thank you!